CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # browser-switch-android Release Notes
 
+## unreleased
+
+* Upgrade Android Gradle Plugin to version 4.0.0
+* Upgrade Gradle to version 6.1.1
```

Comment:
Does this actually impact the merchant integration and need to be included in the CHANGELOG? Or is it just used for the build/distribution side of things?

Reply:
We were thinking it could affect a merchant's CI somewhere, possibly. We added it in for good measure but technically it may be omitted. A good point here because we will probably replicate this convention across the other repos as well.

I'm up for removing these since the Android Gradle Plugin and Gradle version are just dependencies. It would also make CHANGELOG easier to maintain.

Reply:
We will remove these CHANGELOG entries entirely